### DISCUSSION TITLE
Future?

### DISCUSSION THREAD

**Discussion by Manubi** [gqty-dev/gqty#1590] [General]
Hi,
I wanted to try gqty as the idea of not to have to generate the typings is really intriguing and time-saving. 
I saw on the homepage that you are working on V3 but it doesn't seem like it.

So quick question: will this project be continued? If yes, when?

**Comment by boompttn** [comment]
I think they're still working on it. See https://github.com/gqty-dev/gqty/branches/active, especially the `feat/scoped-query` branch

**Comment by vicary** [comment]
We have V3 (the new core) ready in alpha, with many amazing and fantastic users alpha testing with their own projects. When alpha bugs are mostly squashed, it will go to beta followed by a stable release shortly afterwards.

You may track the development of current alpha at #1544, and future roadmaps at #1247.

Our current website at https://gqty.dev has been updated to reflect the V3 functionalities.

You are really welcomed to try out the latest alpha versions, it will surely help speed up the process. If you hit anything, please feel free to report on GitHub and Discord, I'll be there with you.
